metabase/metabase
# Fix for temporal unit parameters in click behaviors

Epic https://github.com/metabase/metabase/issues/42118
Closes https://github.com/metabase/metabase/issues/46069

Fixes an issue with temporal-unit parameters and dashboard click behaviors found during a bugbash.

How to verify:
- Create a question -> Products -> Save
- Create a question -> Orders -> Count -> Breakout by Created At -> Save
- Add both questions to a dashboard
- Add a temporal unit parameter and connect to Orders.CreatedAt
- Click behaviors for the Orders question -> Select any column -> Update dashboard filter -> Select the temporal unit parameter -> Should say that there are no available targets
- Click behaviors for the Products question -> Select Category -> Update dashboard filter -> Select the temporal unit parameter -> Should allow to map -> Save
- Clicking on the Category will now reset the temporal unit parameter because the value is not a valid temporal unit. 
- Valid values would work. To test this, you could create a SQL question that returns strings like `hour`, `day` and use it instead of Products.
